pkg/installer/portal-manager.go

```diff
@@ -60,15 +60,18 @@ func (in *Installer) installPortalManagerClient() error {
 	}
 
 	if err := in.setFieldInFsManifest(filepath.Join(stosDir, portalClientDir, kustomizationFile),
-		buildStringForKustomize(in.stosConfig.Spec.Install.AdminUsername, in.stosConfig.Spec.Install.AdminPassword, in.stosConfig.Spec.Install.PortalAPIURL),
+		buildStringForKustomize(in.stosConfig.Spec.Install.AdminUsername,
```

Comment:
Do we need/have validation to the fields? Empty values doesn't make sense for example.